ParsingModules/BibliographyParsingModule.py

```diff
@@ -0,0 +1,85 @@
+import re
+from Certificate import Certificate
+
+from Enum.JsonStructureKeys import JsonStructureKey
+from JsonItems.BibliographyJson import BibliographyJson
+from ParsingModules.IParsingModule import IParsingModule
+
+
+class BibliographyParsingModule(IParsingModule):
+    _bibliography_title_regex_string = r"[\d\s.]*(?:References|Referenced Literature|Bibliography|" \
+                                       r"REFERENCE DOCUMENTS|Literature)"
+    _bibliography_regex_string = r'\[.{1,20}\]\n* {1,50}[“\w].*\n(?: {1,50}[\(“\w].*(?:\n)+)*'
+
+    @staticmethod
+    def _change_newline(string: str) -> str:
+        change_newlines_to_space = re.sub(r"\n\s+", " ", string)
+        return re.sub("\n", "", change_newlines_to_space)
+
+    @staticmethod
+    def _is_bib_on_page(certificate: Certificate, page_number: int) -> bool:
+        page_content = certificate.get_page_content(page_number)
+        founded = re.search(BibliographyParsingModule._bibliography_title_regex_string, page_content)
+
+        if founded is not None:
+            if re.search(BibliographyParsingModule._bibliography_regex_string,
+                         certificate.get_page_content(page_number)) is None:
+                return False
+            else:
+                return True
+        return False
```

Comment:
```suggestion
return re.search(BibliographyParsingModule._bibliography_title_regex_string, page_content) is not None and \
                   re.search(BibliographyParsingModule._bibliography_regex_string, page_content) is not None
```
This code is unnecessarily obscure and hard to read. It could be simplified to improve the readability.